packages/plugin-core/src/sourcemaps.ts

```diff
@@ -81,7 +82,9 @@ export async function buildBodyForSourcemapUpload(
 ): Promise<FormData> {
   const form = new FormData()
 
-  const minifiedUrl = new URL(sourcemapData.jsFilename, hbOptions.assetsUrl).href
+  const url = new URL(hbOptions.assetsUrl)
```

Comment:
This is strange, the second parameter of URL() constructor is the baseUrl, so it shouldn't be replacing it.

I will test this on a nextjs project as well.

Reply:
I confirm that the second parameter of the URL constructor will strip out any pathnames in the string.